gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -841,23 +842,31 @@ gst_tensorsinfo_compare_to_string (const GstTensorsInfo * info1,
 
   g_return_val_if_fail (info1 != NULL && info2 != NULL, NULL);
 
-  for (i = 0; i < NNS_TENSOR_SIZE_LIMIT; i++) {
+  for (i = 0; i < NNS_TENSOR_SIZE_LIMIT + NNS_TENSOR_SIZE_EXTRA_LIMIT; i++) {
     if (info1->num_tensors <= i && info2->num_tensors <= i)
       break;
 
     if (info1->num_tensors > i) {
-      tmp = gst_tensor_get_dimension_string (info1->info[i].dimension);
-      left = g_strdup_printf ("%s [%s]",
-          gst_tensor_get_type_string (info1->info[i].type), tmp);
+      tmp =
+          gst_tensor_get_dimension_string (gst_tensors_info_get_nth_info (
```

Comment:
line 851, 855, 863, and 868
recommend adding val `GstTensorInfo *_info;` to reduce function calls.

Reply:
I updated those lines, Thanks!